packages/account/src/Sections/Security/Passkeys/components/__tests__/description-container.spec.tsx

```diff
@@ -7,36 +7,34 @@ describe('DescriptionContainer', () => {
         const description_data = [
             {
                 question: 'What are passkeys?',
-                description:
-                    'Passkeys are a security measure that lets you log in the same way you unlock your device: with a fingerprint, a face scan, or a screen lock PIN.',
+                descriptions: [
```

Comment:
i think its better we use same config. we are doing same with other components

Reply:
alright, but what is the sense of test then? if something is changed accidentally in the component, test will notify us.
if we use same config for ui and test - no sense to use tests then, because they will pass anytime right?

Reply:
so basically we can esport any text content to the all tests. but it is wrong I believe 🙏

Reply:
lets discuss internally how can we make it in accounts v2.

Reply:
Since u r using li tag to render description, may I suggest `getAllByRole('listitem')` and counting. That way u need n't test the exact strings